src/03-modality-agnostic-files.md

```diff
@@ -27,6 +27,7 @@ Every dataset MUST include this file with the following fields:
 | EthicsApprovals    | OPTIONAL. List of ethics committee approvals of the research protocols and/or protocol identifiers.                                                                                                                                                                |
 | ReferencesAndLinks | OPTIONAL. List of references to publication that contain information on the dataset, or links.                                                                                                                                                                     |
 | DatasetDOI         | OPTIONAL. The Document Object Identifier of the dataset (not the corresponding paper).                                                                                                                                                                             |
+| HEDVersion         | OPTIONAL. HED version number or the name of HED XML schema file used to validate HED tags for study.                                                                                                                                              |
```

Comment:
If there's a significant likelihood of backwards-incompatible changes, then this should probably be RECOMMENDED:

```suggestion
| HEDVersion         | RECOMMENDED if HED tags are used. HED version number or the name of HED XML schema file used to validate HED tags for study.                                                                                                                      |
```

If it's mostly a way to test new features, then OPTIONAL seems fine.